src/c++/vdo/base/slab-depot.c

```diff
@@ -1223,14 +1223,15 @@ static void reclaim_journal_space(struct slab_journal *journal)
 	WRITE_ONCE(journal->events->flush_count, journal->events->flush_count + 1);
 	if (length < journal->flushing_deadline) {
 		/* Schedule more writes the closer to the deadline we get. */
-		write_count = max_t(block_count_t,
-				    write_count / (journal->flushing_deadline - length + 1),
-				    1);
+		write_count =
+			max_t(block_count_t, (write_count /
+					      (journal->flushing_deadline - length + 1)), 1);
```

Comment:
I think this might look better with the division and "1);" on one line, making it two lines (well, three if you keep the line break after "="), if you don't like the original three-line version. Looks to me like it'll still be under 100 columns.

Reply:
Why yes, I do believe you are correct. I am mirroring what Kine did it change 	2efa17775e4f2ee4f9da99dc160cac6094eebfb0 "dm vdo wait-queue: add proper namespace to interface"
Think it's worth changing?